README.md

```diff
@@ -11,16 +11,34 @@ The service handles device connectivity as well as receiving and storing transmi
 
 ## Configuring and running the service
 
-Register a publicly available endpoint (DNS record) to receive device connections. Tesla devices will rely on a mutual TLS (mTLS) WebSocket to create a connection with the backend. The application has been designed to operate on top of Kubernetes, but it can run as a standalone binary.
-
-See https://developer.tesla.com/docs/fleet-api#setup for setting up and registering a developer account. [check_csr.sh](tools/check_csr.sh) can be used to validate CSR submissions and ensure the account's public key is available.
-
-Configure vehicles for streaming using FleetAPI's [fleet-telemetry-config](https://developer.tesla.com/docs/fleet-api#fleet_telemetry_config-create).  Validate vehicle configs are compatible with your server by running [check_server_cert.sh](tools/check_server_cert.sh) passing the path of your proposed vehicle config:
-```
-./tools/check_server_cert.sh /tmp/proposed_config.json
-```
-
-To troubleshoot issues receiving data from vehicles, see the [fleet_telemetry_errors](https://developer.tesla.com/docs/fleet-api#fleet_telemetry_errors) endpoint.
+### Setup steps
+
+1. Create a third-party application on [developer.tesla.com](https://developer.tesla.com).
+   - In the "Client Details" step, it is recommended to select "Authorization Code and Machine-to-Machine" for most use cases. "Machine-to-Machine" (M2M) should only be selected for business accounts that own vehicles.
+2. Generate a PEM-encoded EC public key using the secp256r1 curve (prime256v1).
```

Comment:
if step 3 is generating public key, should this read `EC key` not `EC public key`?